### ISSUE TITLE
Translations need checking and updating

### ISSUE THREAD

**Issue by blackadderkate** [blackadderkate/weather-widget-2#11]
Translations need checking and updating, as there are a few important additions - especially "Latitude", "Longtitude", "Altitude" and the error checking for those items.

The 2.1.x release needs even more translations adding, because of the search routine.

**Comment by solbu** [comment]
If I may suggest a solution.
I am involved in translation into norwegian at the translation project. https://translationproject.org/
They have translation teams for just about every available language in existence.
Any translation string updates will automatically result in an email going to the various translatior teams with notification on a new po file that need to be updated, and they will update in their own pace.
Some packages send translation updates a week before a planed release, which gives translator a window for updating the strings in their .po files.

As an example, here is a link to a package I maintain, that hosts all its translations on the translation project:  https://translationproject.org/domain/grip.html
Updating the translations are as simple as an rsync command.

If you do opt to host translations there, you can no longer accept any translations or translation updates from Pull Requests. you need to direct any changes to the TP project, and they are very strict on this policy – which I personally agree with.

**Comment by Vistaus** [comment]
Yeah, there are several online options, like Translation Project, Transifex, Weblate, POEditor.com, etc.

**Comment by Vistaus** [comment]
I personally don't like that. I prefer the old-fashioned way or a site where you do need an account, like one of the ones I mentioned. Weblate doesn't even require much - just your email adress and a user name. But at least there's a little bit of control over the translations/teams, unlike Translation Project.

**Comment by blackadderkate** [comment]
My "two cents":

As long as the translations are more accurate than "Google Translate", and somehow end up in this repository when I want to publish a new release, I am very flexible as to how that happens, and who does it.

All I will say is:

Life is short. What I **don't** want is contributors to this widget duplicating each-others work, or wasting anyone's time - including their own! At the end of the day, this is a KDE widget with probably a handful of users, not Windows 11!

Oh, and it might be better to translate using the "placeSearch" branch - this is my main focus going forward (it has a searchable database for Met.No places) and contains more entries to be translated than the "master", and the plan is to convert it into the Master branch soon.

**Comment by eselma** [comment]
Just joined GitHub to contact the developer of this project.
Just got new translations (of the current version, 2.1.6 for me) to Catalan (.po and .mo files). 
I am an amateur translator (and lecturer of Electronics in the Tarragona University) with 20 + years in the job. Of course, I DO NOT use Goggle translate (awful results), but follow the rules of SoftCatalà (platform of Catalan Translators). On the other hand, have accounts at Transifex and Launchpad for other projects.
As I can not push new files to the project, perhaps an easy way could be attaching the .po /.mo files to this message, hope this works.
Many thanks for your nice work.

Eduard Selma.

[translation_CA.zip](https://github.com/blackadderkate/weather-widget-2/files/8056441/translation_CA.zip)

**Comment by blackadderkate** [comment]
Thank you Eduard - I really appreciate you spending your time and effort on this.

I have checked your translations into the repository, but I am just about to logoff for the night so I won't be able to publish a new release containing your translations until tomorrow. :-(

FYI, I _think_ I only need the ca.po file - we have a compiler script that generates the MO files for us, and I think the ca_ES file is not necessary (but I could be wrong about that).

**Comment by eselma** [comment]
Thanks for your prompt response, Kate. Do not worry, there is no really any hurry.
In fact, I compiled (and added the ca_ES, identical) because I am using them in my computers already; as they are small files they could be uploaded to make it easier for you. 
And yes, the ca.po and ca_ES.po are the same file (there are ca_FR and ca_IT versions, that in this case should look the same, too).
Have a good week.
